BUILD.bazel

```diff
@@ -3,15 +3,23 @@ load("//common:browsers.bzl", "chrome_data", "firefox_data")
 load("//java:browsers.bzl", "chrome_jvm_flags", "firefox_jvm_flags")
 load("//java:defs.bzl", "artifact")
 
+# Can only use this if it returns a single file.
+filegroup(
```

Comment:
I added this because `//:license` returns an array of files which can be included